corgi/core/models.py

```diff
@@ -986,6 +970,34 @@ def get_roots(self) -> list[ComponentNode]:
     def build_meta(self):
         return self.software_build.meta_attr
 
+    @property
+    def download_url(self) -> str:
+        """Report a URL for RPMs and container images that can be used in manifests"""
+        if self.software_build and self.software_build.type == SoftwareBuild.Type.BREW:
+            # RPM ex:
+            # /vol/rhel-7/packages/emacs/24.3/23.el7/ppc64le/emacs-common-24.3-23.el7.ppc64le.rpm
+            if self.type == Component.Type.RPM:
+                return (
+                    f"{settings.BREW_DOWNLOAD_ROOT_URL}/vol/"
```

Comment:
BREW_DOWNLOAD_ROOT_URL is not available outside the RH firewall. Therefore we wouldn't want to use it in manifests.

Reply:
Details below - will fix.